kml/serdes_common.hpp

```diff
@@ -21,4 +24,14 @@ std::string PointToString(m2::PointD const & org);
 
 std::string PointToString(geometry::PointWithAltitude const & pt);
 
+namespace
```

Comment:
Anonymous namespace should be used only in cpp files to avoid exporting these symbols to other source files. Here it should be removed.

Reply:
Fixed